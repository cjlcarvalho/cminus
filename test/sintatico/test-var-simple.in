void main(void)
{
    x;
    y;
    arr; /* semantically bad */
    arr[1];
    arr[x];
    arr[x+2];
}
