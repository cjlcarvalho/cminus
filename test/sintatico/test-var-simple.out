[program 
  [decfunc 
    [void]
    [main]
    [paramlist]
    [block [x] [y] [arr] [arr [1]] [arr [x]] [arr 
        [+ [x] [2]]] 
    ]
  ]
]
